### DISCUSSION TITLE
Estimated cost calculation

### DISCUSSION THREAD

**Discussion by jukkapirinen** [visualapproach/WiFi-remote-for-Bestway-Lay-Z-SPA#749] [Q&A] [answered]
How is the estimated cost calculated? If I set the price to 13 or 0,013, price per kWh will be around 7 or 0,07. Am I missing something or is there a bug? 

I check the values from home assistant, so divide total energy price with total energy and it counts 7. Tried to reset the counters in web ui, but still the same after running a while.
Using firmware 2024-06-17-1334,  MIAMI2021.

**Comment by jukkapirinen** [comment]
For example today pump has been running 6 hours between 15-21. Pump looks to be taking 42W.
Total energy has been 11.6kWh-11.35kWh=0.25kWh, that looks correct. 

Energy cost from the same time period 3.64-1.88=1.76

1.76snt/0.25kWh=7.04 snt/kWh


I have set price in web ui to 13.

**Comment by jukkapirinen** [comment]
Quickly testing this seems to fix the issue: 
https://github.com/visualapproach/WiFi-remote-for-Bestway-Lay-Z-SPA/compare/master...jukkapirinen:WiFi-remote-for-Bestway-Lay-Z-SPA:master

So multiply with the time between the measurements.

**Reply by visualapproach** [reply]
Sounds reasonable:)
I'd appreciate if you make the PR to develop branch plz
Thanks for your support/VA

**Reply by visualapproach** [reply, marked as answer]
I uploaded a new version and included this. Thx
